example/testlogger2.cpp

```diff
@@ -163,8 +173,8 @@ testlogger_cli::testThreadedConsoleLogger()
     tobj1->l = loggerCr;
     TestThreadObject2 *tobj2 = new TestThreadObject2(100, UNQL::LOG_DBG, "But *definitely* NOT this...");
     tobj2->l = loggerCr;
-    TestThreadObject2 *tobj3 = new TestThreadObject2(150, UNQL::LOG_DBG, "And NOT even this...");
-    tobj3->l = loggerCr;
+    TestThreadObject2 *tobj3 = new TestThreadObject2(150, UNQL::LOG_INFO, "You *might* see this...");

```

Comment:
- what the '150' means? it would be better to use a named var/const 
- same for above line 174

Reply:
will do